plugins/woocommerce/src/Internal/Integrations/WPConsentAPI.php

```diff
@@ -0,0 +1,94 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\Internal\Integrations;
+
+use Automattic\Jetpack\Constants;
+use Automattic\WooCommerce\Internal\Traits\ScriptDebug;
+use WP_CONSENT_API;
+
+/**
+ * Class WPConsentAPI
+ *
+ * @since x.x.x
+ */
+class WPConsentAPI {
+
+	use ScriptDebug;
+
+	/**
+	 * Register the consent API.
+	 *
+	 * @since 0.1.0
```

Comment:
❔Probably not since v0.1.0 😄 
```suggestion
	 * @since x.x.x
```
Or since the whole class is already documented as `@since x.x.x`, remove this tag altogether